rfc-095-long-term-future-apps.md

```diff
@@ -0,0 +1,85 @@
+# RFC 95: Long term future of applications
+
+## Summary
+
+This RFC describes the direction of travel for the applications that make up GOV.UK.
+
+## Problem
+
+We currently - September 2018 - have 58 separate applications running on GOV.UK. The [Platform architecture principles][princ] explain where we're going from an architectural standpoint. This RFC is an attempt to explain the long term vision of each of GOV.UK's apps.
+
+The purpose of doing this is to make clear:
+
+- where to put new functionality
+- which applications need particular attention
+
+## Proposal
+
+| Name | Description | Rough action plan |
+| -- | -- | -- |
+| calculators | Serves the Child benefit tax calculator on GOV.UK | Merge into smart-answers |
+| calendars | Serves /bank-holidays and /when-do-the-clocks-change on GOV.UK | Merge into custom frontend |
+| ckanext-datagovuk | Extension for use with datagovuk_publish | Retire in favour of datagovuk_find/datagovuk_publish? |
+| collections-publisher | Publishes step by steps, /browse pages, and legacy /topic pages on GOV.UK | While topics and browse pages will be removed, this can still be the app for curating collections. We might move Whitehall's document collections into this app |
+| content-audit-tool | Deprecated application to audit content | Retire, as it's not used |
+| content-performance-manager | Data warehouse that stores content and content metrics to help content owners measure and improve content on GOV.UK | Rename to "data warehouse" |
```

Comment:
I thought this was going to be called `content-data`?

Reply:
Fixed!